camayoc/tests/qpc/cli/conftest.py

```diff
@@ -1,11 +1,7 @@
 """Test utilities for quipucords' ``qpc`` tests."""
-import re
-
-import pexpect
```

Comment:
:tada:

Reply:
Let's file an issue and prioritize beefing up the `make lint` target that the CI runs to make sure we catch unused imports too

Reply:
It already did, it's how I found out about the unused imports :)